src/components/presenters/Artist/ArtistItem.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const ArtistItem = ({ artist, artistId, score, country = 'N/A' }) => {
+  return (
```

Comment:
have you seen the disambiguation key in the API response. you can use that if you want